opm/autodiff/BlackoilModelBase.hpp

```diff
@@ -294,16 +294,33 @@ namespace Opm {
         std::vector<V>
         variableStateInitials(const ReservoirState& x,
                               const WellState& xw) const;
+        void
+        variableReservoirStateInitials(const ReservoirState& x,
+                                       std::vector<V>& vars0) const;
+        void
+        variableWellStateInitials(const WellState& xw,
+                                  std::vector<V>& vars0) const;
 
         std::vector<int>
         variableStateIndices() const;
 
+        std::vector<int>
+        variableWellsStateIndices() const;
```

Comment:
There seems to be a slight inconsistency in the name of this function, would not `variableWellStateIndices` be better?